src/top/run/anchor.py

```diff
@@ -0,0 +1,64 @@
+"""
```

Comment:
I feel like this file should be in `top/model` rather than `top/run`. See the explanation on the package structure at the top.

Reply:
I think I will not use this file because the method of using anchor has changed, but I will move it to that location.